pyqg/model.py

```diff
@@ -118,6 +121,7 @@ def __init__(
         g= 9.81,                    # acceleration due to gravity
         q_parameterization=None,    # subgrid parameterization in terms of q
         uv_parameterization=None,   # subgrid parameterization in terms of u,v
+        parameterization=None,      # subgrid parameterization (type will be inferred)
```

Comment:
I'd prefer to drop this option

Reply:
Per an offline discussion, we're keeping this for now, but considering refactoring this to just take a single `parameterizations` iterable.